python/ray/scripts/scripts.py

```diff
@@ -572,12 +583,29 @@ def start(node_ip_address, address, port, redis_password, redis_shard_ports,
             # not provided.
             num_redis_shards = len(redis_shard_ports)
 
-        if address is not None:
+        if address is not None or external_kv_storage_config is not None:
             cli_logger.print(
                 "Will use value of `{}` as remote Redis server address(es). "
                 "If the primary one is not reachable, we starts new one(s) "
-                "with `{}` in local.", cf.bold("--address"), cf.bold("--port"))
-            external_addresses = address.split(",")
+                "with `{}` in local.", cf.bold("--external-kv-storage-config"),
+                cf.bold("--port"))
+            external_addresses = []
+            if external_kv_storage_config is not None:
+                re_obj = re.match(r"(redis):(.*)", external_kv_storage_config,
+                                  re.M | re.I)
+                if re_obj is None:
+                    raise ValueError(
+                        "--external-kv-storage-config"
+                        "using a format like: redis:host1:port1,host2:port2")
+                else:
+                    external_addresses = re_obj.group(2).split(",")
+            elif address is not None:
+                cli_logger.warning(
+                    "{} will be deprecated,"
```

Comment:
How about:
```
Using --address on head node is deprecated. If you are starting a node to connect to a Ray cluster, please remove the --head option. Otherwise if you want to specify an external Redis address for storage, please use --external-kv-storage-config=redis:...
```